README.md

```diff
@@ -247,7 +303,7 @@ While Equinox is implemented in F#, and F# is a great fit for writing event-sour
 
 4. browse writes at http://localhost:30778/web/index.html#/streams
 
-### Store data in [Azure CosmosDB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
+### Store data in [Azure CosmosDb](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
```

Comment:
all occurrences of 'CosmosDb' term in this doc have been changed to 'CosmosDB' - would be nice ot avoid eggregious diffs